README.md

```diff
@@ -6,25 +6,36 @@ Human-friendly command line tool for the GitHub API.
 
 ### Command Line Tool
 
+#### Install globally (recommmended)
+
+```shell
+npm install --global @financial-times/github
+
+github --help
 ```
-$ npx @financial-times/github --help
-github <command>
-
-Commands:
-  github project:add-pull-request  Add a pull request to a project
-  github project:create            Create a new project
-  github pull-request:create       Create a new pull request
-
-Options:
-  --token    GitHub personal access token                              [string]
-  --version  Show version number                                       [boolean]
-  --help     Show help                                                 [boolean]
+
+When you run the tool it will automatically notify you if there is a newer
+version of it available for you to update to. This
+[can be disabled](https://www.npmjs.com/package/update-notifier#user-settings)
+if you'd prefer not to be notified about updates.
+
+Note: If you install this tool globally on your machine and then run
```

Comment:
I didn't know this! For some reason, I thought the npx would override the globally installed one. Good to know 👍